src/modules/flow/http-server/http-server.c

```diff
@@ -275,11 +275,11 @@ common_handle_response_cb(struct sol_flow_node *node,
     r = type->response_cb(mdata, &response->content, send_json);
     SOL_INT_CHECK_GOTO(r, < 0, err_exit);
 
-    err_r = sol_http_params_add(&response->param, SOL_HTTP_REQUEST_PARAM_HEADER(
+    r = sol_http_params_add(&response->param, SOL_HTTP_REQUEST_PARAM_HEADER(
         HTTP_HEADER_CONTENT_TYPE, (send_json) ? HTTP_HEADER_CONTENT_TYPE_JSON :
         HTTP_HEADER_CONTENT_TYPE_TEXT));
 
-    if (err_r < 0) {
+    if (r < 0) {
         r = -ENOMEM;
```

Comment:
I think we dont need this, because the function already set the r, and maybe you could do something like this instead of this if.

SOL_INT_CHECK_GOTO(r, < 0, err_exit);